sdk/nodejs/src/bucketing.ts

```diff
@@ -1,8 +1,38 @@
 import { instantiate, Exports } from '@devcycle/bucketing-assembly-script'
+import { DVCLogger, DVCReporter } from '@devcycle/types'
+import { DVCOptions } from './types'
 
 let Bucketing: Exports | null
-export const importBucketingLib = async (): Promise<void> => {
-    Bucketing = await instantiate()
+
+export const importBucketingLib = async (
+    { logger, options }:
+    { logger?: DVCLogger, options?: DVCOptions }
```

Comment:
Since both parameters are optional, this can be defaulted to `{}` to simplify a few things.